### PR TITLE
doc: add minutes for meeting 16 jan

### PR DESCRIPTION
**Pull request by gireeshpunathil**

### CODE REVIEW

File: meetings-other/2019-01-16.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* Gireesh Punathil (@gireeshpunathil)
+* Michael Dawson (@mhdawson)
+* Matheus Marchini (@mmarchini)
+* Ruben Bridgewater (@BridgeAR)
+
+## Agenda
+
+## Announcements
+
+* No accouncements this week.
```
Comment (richardlau) [inline meetings-other/2019-01-16.md]:
```suggestion
* No announcements this week.
```

File: meetings-other/2019-01-16.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+*Extracted from **diag-best-practices-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+* Diagnostics Best Practices Guide content definition meeting [#254](https://github.com/nodejs/diagnostics/issues/254)
+
+* Baseline issue to discuss “Best practices” - https://github.com/nodejs/diagnostics/issues/211
+  * First question is how we structure the Best parctices guide.
```
Comment (richardlau) [inline meetings-other/2019-01-16.md]:
```suggestion
  * First question is how we structure the Best practices guide?
```

File: meetings-other/2019-01-16.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    * Michael, first thought is we’d structure around issue type. (ex: symptom)
+    * Matheus, start with symptom then narrow down to deployment scenarios. (same as Michael)
+    * Gireesh, one issue is that there may be a lot of deployment variations so hard to 
+      start there. On the other hand production deployments will still need the steps to get
+      There.
+    * Michael, starting with symptom provides common basis that can be used to build base content
```
Comment (richardlau) [inline meetings-other/2019-01-16.md]:
```suggestion
    * Michael, starting with symptom provides common basis that can be used to build base content.
```

File: meetings-other/2019-01-16.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      Additional level of production level deployment can be built on top of it.
+    * Gireesh, starting with symptom we can also re-use existing documentation more easily.
+    * After we are in discussion we seem to be in agreement with structuring around symptoms
+      and then having a second layer with additional guidance for specific deployment
+      environments.
+
+* Next question is how to prioritize the content
```
Comment (richardlau) [inline meetings-other/2019-01-16.md]:
```suggestion
* Next question is how to prioritize the content?
```

File: meetings-other/2019-01-16.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+* Diagnostics Best Practices Guide content definition meeting [#254](https://github.com/nodejs/diagnostics/issues/254)
+
+* Baseline issue to discuss “Best practices” - https://github.com/nodejs/diagnostics/issues/211
+  * First question is how we structure the Best parctices guide.
+    * Michael, first thought is we’d structure around issue type. (ex: symptom)
+    * Matheus, start with symptom then narrow down to deployment scenarios. (same as Michael)
+    * Gireesh, one issue is that there may be a lot of deployment variations so hard to 
+      start there. On the other hand production deployments will still need the steps to get
+      There.
```
Comment (richardlau) [inline meetings-other/2019-01-16.md]:
```suggestion
      there.
```

**Comment by gireeshpunathil** [comment]
@richardlau - thanks; I am also not sure if there could be other meetings of this class, kept it separately as I think this is different from WG meetings - the proceedings that run up to this meeting, the people that involved etc. However I am open to move the file if there are more (and strong) opinions for it.

**Comment by gireeshpunathil** [comment]
landed as 3f1c93cbf86579563b7c1019caea9b32be1fb937

**Pull request merged**